tests/ui/pages/data_entity/structure.ts

```diff
@@ -0,0 +1,53 @@
+import Button from '../../elements/button';
+import InputField from '../../elements/input-field';
+import DataEntityPage from './data_entity.page';
+
+const SELECTORS = {
+  structureTab: `[role="tab"]:has-text("Structure")`,
+  submitButton: `[type="submit"]`,
+  columnString: name => `.MuiGrid-root.MuiGrid-container.css-1rwljbm:has-text('${name}')`,
+  inputLabel: `[placeholder="Enter label name…"]`,
+  inputDescription: `[name="internalDescription"]`,
+  createNewLabelLink: `[role="presentation"]:has-text('Create new')`,
+};
+
+export default class StructurePage extends DataEntityPage {
+  async editColumn(name: string) {
```

Comment:
`editColumn` - This name is weird. You just click the button in this method. That's why it will be better to name it `clickEditButton`, for instance. 

Please note that we,  probably, will be able to get rid of this method after adding a getter.


UPDATE: 

I just want to remind that this button is placed in the list with results. (See screenshot: https://ibb.co/G28JMz7)
That's why I propose to re-implement this method in another way.

1. In the list with results find the list item which contains the name that you need (Note: you should be able to use the following method for this: https://github.com/opendatadiscovery/odd-platform/blob/5372d6c/tests/ui/elements/list.ts#L79)
1. in list item from the previous step find "Edit" button and click